python/dolfinx/wrappers/la.cpp

```diff
@@ -178,6 +173,11 @@ void declare_objects(nb::module_& m, const std::string& type)
 template <typename T>
 void declare_functions(nb::module_& m)
 {
+  m.def(
+        "norm",
+        [](const dolfinx::la::Vector<T>& x, dolfinx::la::Norm type)
+        { return dolfinx::la::norm(x, type); },
+        "vector"_a, "type"_a = dolfinx::la::Norm::l2);
```

Comment:
Remove the default value here - it's already set in the Python wrapper.

Best to avoid default values in the nanobind layer the reduce the magic.